src/types.db

```diff
@@ -14,7 +14,7 @@ cache_eviction		value:DERIVE:0:U
 cache_operation		value:DERIVE:0:U
 cache_ratio		value:GAUGE:0:100
 cache_result		value:DERIVE:0:U
-cache_size		value:GAUGE:0:U
+cache_size		value:GAUGE:0:1125899906842623
```

Comment:
I picked that value as it was used in other places where you need a 64bit
counter....



On Wed, Sep 2, 2015 at 6:01 PM Brian O'Neill <btoneill@gmail.com> wrote:

> The cache size value of U is a 32bit unsigned int. If you have a large
> memory system, you overrun it and get no values. We ran into that with a
> system that had 256G of memory.
>
>
>
> On Wed, Sep 2, 2015 at 4:45 PM Marc Fournier <notifications@github.com>
> wrote:
>
>> In src/types.db
>> <https://github.com/collectd/collectd/pull/1205#discussion_r38588733>:
>>
>> > @@ -14,7 +14,7 @@ cache_eviction		value:DERIVE:0:U
>> >  cache_operation		value:DERIVE:0:U
>> >  cache_ratio		value:GAUGE:0:100
>> >  cache_result		value:DERIVE:0:U
>> > -cache_size		value:GAUGE:0:U
>> > +cache_size		value:GAUGE:0:1125899906842623
>>
>> Why change this ?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/collectd/collectd/pull/1205/files#r38588733>.
>>
>